front/components/vaults/VaultWebsiteModal.tsx

```diff
@@ -180,13 +177,7 @@ export default function VaultWebsiteModal({
     }
 
     // Validate Name
-    const nameExists = dataSources.some(
-      // TODO(DATASOURCE_SID): This line was kind of buggy because dataSourceId and sId was name but
-      // dataSourceId was not defined when it mattered (creation) and this not used during patching.
-      // (currently replacing back to .name the real real value for now showing that the line makes
-      // no sense). Will move to sId in subsequent PR.
-      (d) => d.name === dataSourceName && d.name !== dsName
-    );
+    const nameExists = dataSources.some((d) => d.name === dataSourceName);
```

Comment:
not sure I get the change of logic here --- before, nameExists was false if the name was the current datasourceView name; now it's true in that case (and will err below with `nameError`). Is that expected?

Reply:
As per the comment the previous implementation didn't make any sense